05-expressions.tex

```diff
@@ -220,6 +220,12 @@ \section{for}
 
 Variable \expr{v} is then available within loop body \expr{e2} and holds the value of the individual elements of collection \expr{e1}.
 
+The closest Haxe for-loops come to C-style for-loops is probably using the range operator \expr{...}. It is a binary operator taking two \expr{Int} arguments: \expr{min...max} returns an \expr{IntIterator} instance that iterates from \expr{min} (inclusive) to \expr{max} (exclusive). Note that \expr{max} may not be smaller than \expr{min}.
```

Comment:
I think we should change both to operands. We should change the "return" too but I don't really know to what, so let's just leave that.